### PR TITLE
Docs additions, field configuration

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/guide/en/field-configuration.md [hunk 1]
```diff
@@ -1,33 +1,175 @@
...
+- [`inputContainerAttributes`](field-methods.md#inputcontainerattributes--addinputcontainerattributes) - HTML attributes
+for outer container that wraps the input.
+- [`inputContainerClass`](field-methods.md#inputcontainerclass--addinputcontainerclass) - HTML class for outer container
+that wraps the input.
+- [`labelClass`](field-methods.md#labelclass--addlabelclass) - HTML class for label.
+- [`labelConfig`](field-methods.md#labelconfig) - Config with [definitions](https://github.com/yiisoft/definitions) for
+`Label` widget.
+- [`hintClass`](field-methods.md#hintclass--addhintclass) - HTML class for hint.
+- [`hintConfig`](field-methods.md#hintconfig) - config with [definitions](https://github.com/yiisoft/definitions) for
+- `Hint` widget.
```
Comment (vjik) [inline docs/guide/en/field-configuration.md:39]:
```suggestion
```

File: docs/guide/en/field-configuration.md [hunk 2]
```diff
@@ -1,33 +1,175 @@
...
+- [`errorClass`](field-methods.md#errorclass--adderrorclass) - HTML class for error.
+- [`errorConfig`](field-methods.md#errorconfig) - config with [definitions](https://github.com/yiisoft/definitions) for
+- `Error` widget.
```
Comment (vjik) [inline docs/guide/en/field-configuration.md:42]:
```suggestion
```

File: docs/guide/en/field-configuration.md [hunk 3]
```diff
@@ -1,33 +1,175 @@
...
+        'defaultTheme' => 'vertical',
+        'validationRulesEnricher' => new MyValidationRulesEnricher(),
+    ],
+    // ...
+];
+```
+
+No built-in themes and validation rules enricher are used by default.
+
+## Themes' preview
```
Comment (vjik) [inline docs/guide/en/field-configuration.md:155]: Move this to internals.

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ The result of executing the code above will be:
 
 ## Documentation
 
-- [Guide](docs/guide/en/README.md)
```
[the remark this answers is not in the document]
Comment (arogachev) [reply README.md:109]: It's considered as a default index file by Github when viewing a folder. Technically it's the same.

File: docs/guide/en/field-configuration.md (hunk 3 above)
Comment (arogachev) [reply docs/guide/en/field-configuration.md:155]: Why? Is preview for developers only? I think it can be useful for users too.

Comment (vjik) [reply docs/guide/en/field-configuration.md:152]: Yes. It is for developers only.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vjik) [reply README.md:109]: Not same. Without README.md files displays.

[the remark this answers is not in the document]
Comment (arogachev) [reply README.md]: OK. I reverted this change.

File: docs/guide/en/field-configuration.md (hunk 3 above)
Comment (arogachev) [reply docs/guide/en/field-configuration.md]: OK. I moved it to internals.

**Review by samdark — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Comment by samdark** [comment]
:+1:

**Pull request merged**